packages/shared/src/errors/utils/errorUtils.ts

```
@@ -196,7 +197,30 @@ function getCurrentCallStack() {
+function getCurrentCallStack() {
+  const e = new Error();
+  const stack = e.stack;
+  return stack;
+}
+
+function logCurrentCallStack(name?: string) {
+  if (process.env.NODE_ENV !== 'production') {
+    if (
+      console &&
+      console.groupCollapsed &&
+      console.groupEnd &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)